File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/non-vul/code_before43.c
// From cwe-snippets, snippets_90/non-compliant/C/cwe-0252/char_sscanf_01.c

static void func()
{
    {
        char dataBuffer[100] = "";
        char * data = dataBuffer;
        if (sscanf(SRC, "%99s\0", data) == EOF)
        {
            printLine("sscanf failed!");
        }
    }
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/non-vul/code_before43.java
// From cwe-snippets, snippets_90/non-compliant/Java/cwe-0382/Servlet_Runtime_01.java

private void func(HttpServletRequest request, HttpServletResponse response) throws Throwable
{
    response.getWriter().write("You cannot shut down this application, only the admin can");

}